packages/apps/job-launcher/server/src/common/constants/index.ts

```diff
@@ -23,7 +23,7 @@ export const MAINNET_CHAIN_IDS = [
 export const SENDGRID_API_KEY_REGEX =
   /^SG\.[A-Za-z0-9-_]{22}\.[A-Za-z0-9-_]{43}$/;
 
-export const HEADER_SIGNATURE_KEY = 'human-signature';
+export const HEADER_SIGNATURE_KEY = 'Human-Signature';
```

Comment:
We have it in lower case in all oracles, let's keep it 
```suggestion
export const HEADER_SIGNATURE_KEY = 'human-signature';
```

Reply:
changed